graphistry/plugins/splunkconn.py

```diff
@@ -0,0 +1,107 @@
+import splunklib.client as client
+import splunklib.results as results
+import sys
+import pandas as pd
+
+class SplunkConnector:
```

Comment:
good we're finally doing this officially

we can probably make a mixin w/ fluent use

```
graphistry.splunk(**CREDS).splunk("my query", mode="nodes"|"edges"|"hypergraph", **OPTS).plot()
```